File: rest/accounts/list-get-example-2/list-get-example-2.5.x.cs
// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Twilio;
using Twilio.Rest.Api.V2010;
using System.Linq;

class Example
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Auth Token at twilio.com/console
        // To set up environmental variables, see http://twil.io/secure
        const string accountSid = Environment.GetEnvironmentVariable("TWILIO_ACCOUNT_SID");
        const string authToken = Environment.GetEnvironmentVariable("TWILIO_AUTH_TOKEN");
        TwilioClient.Init(accountSid, authToken);

        var readOptions = new ReadAccountOptions{
            Status = AccountResource.StatusEnum.Active };
        var accounts = AccountResource.Read(readOptions);

        foreach (var account in accounts) {
            Console.WriteLine(account.DateCreated);
        }
    }
}

File: rest/outgoing-caller-ids/list-get-example-2/list-get-example-2.5.x.js
// Download the Node helper library from twilio.com/docs/node/install
// These consts are your accountSid and authToken from https://www.twilio.com/console
// To set up environmental variables, see http://twil.io/secure
const accountSid = process.env.TWILIO_ACCOUNT_SID;
const authToken = process.env.TWILIO_AUTH_TOKEN;
const client = require('twilio')(accountSid, authToken);

client.api
  .accounts(accountSid)
  .outgoingCallerIds // filter callerids to include only those that have the following number
  .each({ phoneNumber: '+12349013030' }, callerId => console.log(callerId.sid));
